addons/mail/models/mail_thread.py

```diff
@@ -1734,7 +1738,7 @@ def _message_post_process_attachments(self, attachments, attachment_ids, message
                 content = content.encode('utf-8')
             data_attach = {
                 'name': name,
-                'datas': base64.b64encode(content),
+                'datas': base64.b64encode(content or ""),
```

Comment:
If we have nothing to encode shouldn't we avoid creating void attachments instead ?